spec/components/date_range_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe 'date_range', type: :helper do
+  subject do
+    Nokogiri::HTML.parse(date_range)
+  end
+
+  let(:form) do
+    helper.bootstrap_form_for :foo, url: '/' do |f|
+      return f
```

Comment:
One could also argue, though, that passing the form object to a local variable first and returning this from the let block, assumes that `bootstrap_form_for` doesn't change the object after the block. The interesting question now is, would this also happen with the `return`?